taesch/lib/view/custom_widget/add_item_dialog.dart

```diff
@@ -23,6 +24,16 @@ class _AddItemDialogState extends State<AddItemDialog> {
     logger.log(
         level: LogLevel.info,
         logMessage: LogMessage(message: "entered add item dialog"));
+    var tagEditController = TextEditingController();
+    tagEditController.addListener(() => {
+      if (tagEditController.text.endsWith(' ')||tagEditController.text.endsWith(',')) {
```

Comment:
Die Logik des Überprüfen des Strings sollte im ViewModel erfolgen. Diese Methode würde dann einen Wert zurückgeben, mit dem hier weiter gearbeitet werden kann.

Reply:
solved as of 0305b0360d3435e582b2b804c6d990b6fdbeb328